include/seqan3/alphabet/nucleotide/nucl16.hpp

```diff
@@ -315,9 +333,140 @@ constexpr nucl16 nucl16::W{internal_type::W};
 constexpr nucl16 nucl16::Y{internal_type::Y};
 constexpr nucl16 nucl16::UNKNOWN{nucl16::N};
 
+} // namespace seqan3
+
+namespace seqan3::detail
+{
+
+//!\brief seqan3::nucl16 is defined as being a nucleotide alphabet.
+template <>
+struct is_nucleotide<nucl16> : public std::true_type
+{};
+
+} // namespace seqan3::detail
+
 #ifndef NDEBUG
-static_assert(alphabet_concept<nucl16>);
-static_assert(detail::internal_alphabet_concept<nucl16>);
+static_assert(seqan3::alphabet_concept<seqan3::nucl16>);
+static_assert(seqan3::nucleotide_concept<seqan3::nucl16>);
 #endif
 
+// ------------------------------------------------------------------
+// containers
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+    /*!\name Alphabet aliases
+     * \{
+     * \brief Other names (typedefs) for seqan3::nucl16
+     * \relates nucl16
+     */
+    using dna16 = nucl16;
+    using rna16 = nucl16;
+    using dna = nucl16;
+    using rna = nucl16;
+    //!\}
```

Comment:
`/*` is a multi-line comment, `//` is a single line comment. But for doxygen it has a different meaning also. In this case we need multi-line comments in the beginning, because the commands refer to the group and not the member. If you use single-line commands they instead refer to the individual using lines.
I know, it's not really great :neutral_face: